src/test/resources/application.properties

```diff
@@ -1,7 +1,6 @@
 uk.gov.companieshouse.item-handler.health = /healthcheck
 
 spring.kafka.consumer.bootstrap-servers = ${spring.embedded.kafka.brokers}
-spring.kafka.consumer.auto-offset-reset = earliest
```

Comment:
I note you seem to be changing more than one thing at once here?

Reply:
This change didn't have any effect on `test-analyse` build runs. Removing it to ensure it doesn't get carried over unnecessarily. I've updated the PR description.